packages/react-components/src/components/MessageThread.tsx

```diff
@@ -292,11 +292,17 @@ const memoizeAllMessages = memoizeFnAll(
     onRenderMessageStatus:
       | ((messageStatusIndicatorProps: MessageStatusIndicatorProps) => JSX.Element | null)
       | undefined,
-    defaultStatusRenderer: (status: MessageStatus) => JSX.Element,
+    defaultStatusRenderer: (
+      status: MessageStatus,
+      messageThreadReadCount: number,
+      messageThreadParticipantCount: number
+    ) => JSX.Element,
     defaultChatMessageRenderer: (message: MessageProps) => JSX.Element,
     strings: MessageThreadStrings,
     _attached?: boolean | string,
     statusToRender?: MessageStatus,
+    messageThreadReadCount?: number,
+    messageThreadParticipantCount?: number,
```

Comment:
Do we take into account anywhere the fact read receipts stop only work up to 20 participants?
https://docs.microsoft.com/en-us/azure/communication-services/concepts/chat/concepts#:~:text=readReceiptReceived%20-%20when%20another%20participant%20sends%20a%20read,thread%20is%20deleted%20by%20a%20Communication%20Services%20user.
![image](https://user-images.githubusercontent.com/2684369/154140810-495d040c-aa36-45c0-8e92-e76275df9da5.png)

Reply:
Or better question - what is expected to be shown when there are more than 20 participants?